earth_enterprise/src/common/performancelogger.cpp

```diff
@@ -42,18 +162,25 @@ void PerformanceLogger::log(
   stringstream message;
 
   message.setf(ios_base::fixed, ios_base::floatfield);
-  message << setprecision(9)
+  message << setprecision(9)/*
           << operation << " " << object << ": "
           << "start time: " << startTime
           << ", "
           << "end time: " << endTime
           << ", "
           << "duration: " << duration
           << ", "
-          << "thread: " << tid;
+          << "thread: " << tid*/;
+  message << operation << ','
+          << object    << ','
+          << startTime << ','
+          << endTime   << ','
+          << duration  << ','
+          << tid       << ',';
+
   if (size > 0) {
```

Comment:
This is not going to produce a valid CSV.  Every line would need to have the same number of fields.

Reply:
Fixing before merge, just had this moved over from previous version.